### PR TITLE
chore(www): Switch to gatsby-design-tokens package

### PR DESCRIPTION
**Pull request by fk**
Following up on #15596/#15779, this 

* adds the new `gatsby-design-tokens` package (and pins it at `0.0.1`—smelling a `0.0.2` already)
* removes `www/src/utils/tokens`
* rewires things for `gatsby-design-tokens`
* orders imports in `www/src/utils/presets.js` alphabetically (b/c OCD)

### CODE REVIEW

File: www/package.json [hunk 1]
```diff
@@ -22,6 +22,7 @@
     "emotion-theming": "^10.0.10",
     "fuse.js": "^3.2.0",
     "gatsby": "^2.11.0",
+    "gatsby-design-tokens": "0.0.1",
```
Comment (DSchau) [inline www/package.json]: Why pin?

Comment (sidharthachatterjee) [reply www/package.json]:
```suggestion
    "gatsby-design-tokens": "^0.0.1",
```

Comment (sidharthachatterjee) [reply www/package.json]: Spoke to @fk about this. He prefers pinning here to ensure publishing design changes to `www` are deliberate.

**Comment by fk** [comment]
Thanks Dustin and Sid!

**Pull request merged**